html/rightlinebody.html

```diff
@@ -15,8 +15,149 @@
+                                                                    We have recruited 29 new members from 170 total 
+                                                                    applicants this year! Thank you to our Executive board 
+                                                                    for spending several hours for recruitment! We are 
+                                                                    excited to work with our neew members! We completed 
+                                                                    our first events with our new members, here are 
+                                                                    pictures from All Hands 2:
+                                                                </p>
+                                                            </td>
+                                                        </tr>
+                                                    </table>
+                                                </td>
+                                            </tr>
+                                            <tr>
+                                                <td class="sk-td sk-entry">
+                                                    <table class="sk-table">
+                                                        <tr>
+                                                            <td class="sk-td sk-img-right">
+                                                                <img class="sk-img" src ="https://drive.google.com/uc?export=view&id=1TPYVjSS2QwlWlr6Xe8oiaAvLKFQX-JJC">
+                                                            </td>
+                                                        </tr>
+                                                    </table>
+                                                </td>
+                                            </tr>
+                                            <tr>
+                                                <td class="sk-td sk-entry">
+                                                    <table class="sk-table">
+                                                        <tr>
+                                                            <td class="sk-td sk-img-right">
+                                                                <img class="sk-img" src ="https://drive.google.com/uc?export=view&id=18JqJmYshovA9u3e0HCikV9NVD9wpDVWJ">
+                                                            </td>
+                                                        </tr>
+                                                    </table>
+                                                </td>
+                                            </tr>
+                                        </table>
+                                    </td>
+                                    <td class="sk-td" style="padding-right: 30px; padding-top: 85px;">
```

Comment:
are we refactoring all of the inline styles out?

Reply:
yeah that will probably be better for the generating the template, will make the changes